### ISSUE TITLE
Alien Breed Tower Assault CD32 - 'More compatible' & '24-bit addressing' bug

### ISSUE THREAD

**Issue by blinkydoos** [midwan/amiberry#104]
If '24-bit addressing' is ticked then the CD doesn't boot. If you turn the option off then it boots fine. There must be something wrong with '24-bit addressing' as the CD32 has a 68EC020. (Tested on SDL2 version).

**Comment by CypherXG** [comment]
Sry, works by me without any Problems... (SDL1 Version)
I see all, the Intro and the Game....
CPU 68020
FPU none
Speed 14Mhz
Chipset AGA
Blitter normal
Fast Copper off
Collisionlevel Sprite vs Sprite

24-bit adressing ist on
JIT is off or on....no matter
Chipmem 2MB all other memory off....

**Comment by blinkydoos** [comment]
I looked into this some more and there are more strange things happening. If you load the built-in CD32 config, the option 'More compatible' is ticked but this option is only available on 68010 and less. CD's boot fine with this config though (except Alien Breed Tower Assault, I had to turn off 24-bit addressing and it booted).

If you load Amiberry and manually setup a CD32 config and 'More compatible' is not ticked and greyed out then CD's don't boot.

So I think the built-in config's need updating and the options 'More compatible' and '24-bit addressing' need looking into.

(Tested on SDL2 version).
